server.js

```diff
@@ -0,0 +1,140 @@
+var fs = require('fs');
+var path = require('path');
+var bodyParser = require('body-parser');
+var jsonParser = bodyParser.json();
+var express = require('express');
+var app = express();
+var datastore = './data';
+
+function renderJSON(res, data) {
+    res.status(200)
+        .set('Content-Type', 'application/json')
+        .json(data);
+}
+
+function filterJSONFiles(data) {
+    return data.filter(function(file) {
+        return (file.substr(-5) === '.json') ? 1 : 0;
+    });
+}
+
+app.route('/elements')
```

Comment:
You should modularize this (this should go into another file under a routes folder)